string ACTION_NAME=/oni_vicon_player/Open

string record_path
---
string message
---
bool open
int64 progress
int64 progress_max
float64 total_time
uint32 total_vicon_frames
uint32 total_depth_sensor_frames
